R-Interface/jaspResults/src/jaspObject.cpp

```diff
@@ -415,6 +427,25 @@ void jaspObject::setDeveloperMode(bool developerMode)
 	_developerMode = developerMode;
 }
 
+bool jaspObject::connectedToJaspResults()
+{
+
+	// we could add a private bool to jaspObject to shortcircuit this recursive chain
```

Comment:
And then having to update that and all that?
And waste a whole byte for what is probably a few calls up (who would be nesting jaspObjects so deep that it would ever exceed like 5 or so?)

Embrace the recursion ;)